ruma-federation-api/src/transactions/send_transaction_message.rs

```diff
@@ -1,3 +1,4 @@
 //! Endpoint to send live activity messages to another server.
 
+pub mod edu;
```

Comment:
This should probably just be a top-level module in `ruma-federation-api`.

Reply:
Or at least be moved one level up in the module structure if it's only relevant to transactions.